docs/pages/world/_meta.js

```diff
@@ -13,4 +13,7 @@ export default {
   "world-table-illustration": {
     display: "hidden",
   },
+  "table-immutable": {
+    display: "hidden",
+  },  
```

Comment:
No, we don't. But for clarity's sake, I prefer to keep things that are just in one or two related places with the content, rather than as separate components.

Reply:
I'd rather we import it from a separate location that isn't page specific so we don't need this workaround

Reply:
I removed the `_meta.js` lines.